lib/httpserver/httpserver_test.go

```diff
@@ -3,7 +3,11 @@ package httpserver
 import (
 	"encoding/json"
 	"net/http"
+	"net/http/httptest"
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Thanks for tests, but we do not use 3rd party libs for testing purposes. Is it possible to use standard libs only?

Reply:
Hi @hagen1778  Sorry for late response, i was on vacation and did not see this message. I will check if this can be achieved using standard libs.

Reply:
@hagen1778  I have removed the 3rd party libs in the test